src/main/kotlin/com/terraformation/backend/customer/api/OrganizationsController.kt

```diff
@@ -61,27 +61,17 @@ class OrganizationsController(
   @GetMapping
   @Operation(
       summary = "Lists all organizations.",
-      description = "Lists all organizations the user can access.",
+      description = "Lists all organizations the user can access through organization roles.",
   )
   fun listOrganizations(
       @RequestParam("depth", defaultValue = "Organization")
       @Schema(description = "Return this level of information about the organization's contents.")
       depth: OrganizationStore.FetchDepth,
   ): ListOrganizationsResponsePayload {
-    val currentUser = currentUser()
     val elements =
         organizationStore.fetchAll(depth).map { model ->
-          OrganizationPayload(
```

Comment:
Since we are using the AcceleratorOrganizationsController, we no longer need the TF Contact user to also be passed in with the regular organizations list.